src/bundles/SellerRegistration/components/Finish.js

```diff
@@ -1,24 +0,0 @@
-import React from 'react';
```

Comment:
i wouldn't delete this page just yet - we may have to re-enable if things go wrong